tests/test_nlp.py

```diff
@@ -161,11 +161,10 @@ def test_valid_input(self):
 
         inputs = {"data": data}
         bpayload = json.dumps({"inputs": inputs}).encode("utf-8")
-        normalized_inputs, processed_params = normalize_payload_nlp(
-            bpayload, "tabular-classification"
-        )
-        self.assertEqual(processed_params, {})
-        self.assertEqual(inputs, normalized_inputs)
+        for task in ["tabular-classification", "tabular-regression"]:
```

Comment:
also, why touching this file?

Reply:
I got the impression that since tabular classification is tested here and now we add tabular regression, it should be tested in the same way.

Reply:
I agree, but I'd put it in a separate PR.

Reply:
Reverted.